### PR TITLE
Feature: PostCSS import

### PR DESCRIPTION
**Pull request by marcobiedermann**
Add style field to `package.json` and add section to readme.

> When importing a module, it will look for index.css or file referenced in `package.json` in the `style` or `main` fields. You can also provide manually multiples paths where to look at.
[postcss-import](https://github.com/postcss/postcss-import)

### CODE REVIEW

**Review by arcticicestudio — CHANGES_REQUESTED** [review]
Thanks for your contribution :+1: 

Nice addition to simplify the usage/import. I've saw the field in some other projects, but never really investigated time to check which package makes use of this custom field :smile: 

Added some small change requests, but everything else looks fine.

File: README.md [hunk 1]
```diff
@@ -88,6 +88,12 @@ body {
 }
 ```
 
+When using [postcss-import](https://github.com/postcss/postcss-import) you can simply import the CSS file like any JavaScript module:
```
Comment (arcticicestudio) [inline README.md]: Please change the *inline* link to the GitHub repository to a *reference* link and place it at the bottom of the document. You can name it `postcss-import-gh` to adapt to the naming style of the other links.

Comment (arcticicestudio) [inline README.md]: Can you also please add the new paragraph to the documentation in the *Usage* chapter ([`docs/getting-started/usage.md`][ref])

[ref]: https://github.com/arcticicestudio/nord/blob/develop/docs/getting-started/usage.md

**Comment by marcobiedermann** [comment]
You are very welcome. Yeah, I did not know of this either a couple of weeks ago.

> Added some small change requests, but everything else looks fine.

Sorry but I can not find any [reviews](https://github.com/arcticicestudio/nord/pull/88#pullrequestreview-89090013)

**Comment by arcticicestudio** [comment]
Sorry, I've posted the reviews after the first comment, refresh the page to see them.

**Comment by marcobiedermann** [comment]
I am really sorry but I can not find any pending requests / reviews from you on the PR:
image
Clicking on "View Changes" lead to https://github.com/arcticicestudio/nord/pull/88/files/604d3dba836178ba4aa90eef1be9a46a19fe4a05 which are the changes I made.
Even if I would, I can not approve my modifications because this can not be done from the same owner.
Either am I overlooking something obvious or have to review and request some changes again

**Review by arcticicestudio — CHANGES_REQUESTED** [review]

**Comment by arcticicestudio** [comment]
Alright, I forgot to submit the pending change requests after adding the comments :upside_down_face:

**Comment by marcobiedermann** [comment]
Haha, I thought for a second I am really overlooking something.
Fixed in https://github.com/arcticicestudio/nord/pull/88/commits/be02cb278e38589dfd9cce0cbcc686d01ee050da.
If there is anything else I can do for you, feel free to hit me up :)

**Review by arcticicestudio — APPROVED** [review]
Looks fine now, thanks for the changes :+1:

**Pull request merged**